modules/ftrackIdSystem.md

```diff
@@ -0,0 +1,90 @@
+
+**ACCURATE**  
+FTrack’s broad adoption combined with the maturity of the models (6+ years old) gives Flashtalking the global scale with which to maintain a high degree of model resolution and accuracy.
+
+**DURABLE**  
+As new IDs start to proliferate, they will serve as new incremental signals for our models.
+
+---
+
+### Support or Maintenance:
+
+Questions? Comments? Bugs? Praise? Please contact FlashTalking's Prebid Support at [prebid-support@flashtalking.com](mailto:prebid-support@flashtalking.com)
+
+---
+
+### FTrack User ID Configuration
+
+The following configuration parameters are available:
+
+```javascript
+pbjs.setConfig({
+  userSync: {
+    userIds: [{
+      name: 'FTrack',
+      params: {
+        url: 'https://d9.flashtalking.com/d9core' // required, if not populated ftrack will not run
+      },
+      storage: {
+        type: 'html5',           // "html5" is the required storage type
+        name: 'FTrackId',        // "FTrackId" is the required storage name
+        expires: 90,             // storage lasts for 90 days
+        refreshInSeconds: 8*3600 // refresh ID every 8 hours to ensure it's fresh
+      }
+    }],
+    auctionDelay: 50             // 50ms maximum auction delay, applies to all userId modules
+  }
+});
+```
+
+
+
+
+| Param under userSync.userIds[] | Scope | Type | Description | Example |
+| :-- | :-- | :-- | :-- | :-- |
+| name | Required | String | The name of this module: `"FTrack"` | `"FTrack"` |
+| storage | Required | Object | Storage settings for how the User ID module will cache the FTrack ID locally | |
+| storage.type | Required | String | This is where the results of the user ID will be stored. FTrack **requires** `"html5"`. | `"html5"` |
+| storage.name | Required | String | The name of the local storage where the user ID will be stored. FTrack **requires** `"FTrackId"`. | `"FTrackId"` |
+| storage.expires | Optional | Integer | How long (in days) the user ID information will be stored. FTrack recommends `90`. | `90` |
+| storage.refreshInSeconds | Optional | Integer | How many seconds until the FTrack ID will be refreshed. FTrack strongly recommends 8 hours between refreshes | `8*3600` |
+
+**ATTENTION:** As of Prebid.js v4.14.0, FTrack requires `storage.type` to be `"html5"` and `storage.name` to be `"FTrackId"`. Using other values will display a warning today, but in an upcoming release, it will prevent the FTrack module from loading. This change is to ensure the FTrack module in Prebid.js interoperates properly with the [FTrack](https://www.flashtalking.com/identity-framework#FTrack) and to reduce the size of publishers' first-party cookies that are sent to their web servers. If you have any questions, please reach out to us at [prebid-support@flashtalking.com](mailto:prebid-support@flashtalking.com).
```

Comment:
why is this in here? you weren't even a module in 4.14.0. Please do not just copy and paste from [other modules](https://github.com/prebid/Prebid.js/blob/master/modules/id5IdSystem.md)

Reply:
Removed. Guilty.